\documentclass[11pt]{article}
\pdfoutput=1
\usepackage{simplemargins}
\usepackage[pdftex]{graphicx}
\graphicspath{{figures/}}
\usepackage{siunitx}
\bibliographystyle{plain}
\setlength{\parindent}{0pt} 
\setlength{\parskip}{1.6ex} 
\setallmargins{1in} 
\linespread{1.6}

\begin{document}


% Title must be 150 characters or less
\begin{flushleft} 
{\Large \textbf{The genetic architecture of local adaptation I: The genomic landscape of 
foxtail pine (\textit{Pinus balfouriana} Grev. \& Balf.)}}
% Insert Author names, affiliations and corresponding author email.
\\
Christopher J.\ Friedline$^{1}$, 
Brandon M. Lind$^{1}$,
Erin M. Hobson,$^{1}$,
Douglas E. Harwood$^{1}$, 
Annette Delfino Mix$^{2}$,
Patricia E. Maloney$^{3}$, and
Andrew E. Eckert$^{1,4}$
\\
\bf{1} Department of Biology, Virginia Commonwealth University, Richmond, VA 23225
\\
\bf{2} Institute of Forest Genetics, USDA Pacific Southwest Research Station, Placerville, 
CA 95667
\\
\bf{3} Department of Plant Pathology, University of California, Davis, CA 95616
\\
\bf{4} Author for Correspondence

$\ast$ E-mail: aeckert2@vcu.edu
\end{flushleft}

\section{Abstract}

\section{Introduction}
The genetic architecture of phenotypic traits affecting fitness has been a major focus of
geneticists for over a century. Interest in this architecture stems from the interest in explaining
the nature of genetic variation which contributes to adaptive evolution. Architecture in this 
instance refers to the number, type, genomic position, effect size, interactions, and environmental dependency 
of the loci contributing to phenotypic variation that alters fitness. Early efforts to understand this architecture 
focused primarily on the number and effect size of the loci underlying phenotypic variation (e.g. Fisher 1918). 
Recent work has extended this line of research, with myriad studies linking phenotypic with genetic variation through 
some form of linkage mapping, either within pedigrees (XXXXX) or within unstructured populations (XXXX). Despite great
advances in experimental technology, the focus has remained on the number, effect size, and type of loci
contributing to adaptive evolution.

\section{Materials and Methods}

\subsection{Focal species}
Foxtail pine (\textit{Pinus balfouriana}) is five needle species of pine classified into 
subsection \textit{Balfourianae}, section \textit{Parrya}, and subgenus \textit{Strobus} 
(Gernandt et al. 2008). It is one of three species within subsection \textit{Balfourianae} 
(Bailey 1970) and generally is regarded as the sister species to Great Basin bristlecone pine (\textit{P. longaeva}; see 
Eckert and Hall 2006). The natural range of foxtail pine encompasses two 
regional populations located within California that are separated by approximately 500 km - 
the Klamath Mountains of northern California and the Sierra Nevada of southern California 
(Figure 1). These regional populations diverged approximately one million years ago (mya), 
with current levels of gene flow between these regional populations approximately zero 
(Eckert et al. 2008). Within each regional population, levels of genetic diversity and the 
degree of differentiation among local stands differ, with genetic diversity being highest in 
the southern Sierra Nevada population and genetic differentiation being the highest in the 
Klamath population (Oline et al. 2000; Eckert 2006; Eckert et al. 2008).

\subsection{Sampling}
Seed lots from approximately 140 maternal trees distributed throughout the natural range 
of foxtail pine were obtained during 2011 and 2012. XXX of these were sampled from the 
Klamath region, while XXX were sampled from the southern Sierra Nevada region. Approximately 
50 seeds were germinated from each seed lot and 35 of those 50 seedlings were planted in a 
common garden located at the USDA Institute of Forest Genetics, Placerville, California. The 
common garden was established using a randomized block design XXX MORE HERE. Five of the 
140 maternal trees were selected at random ($n = 3$ from the Klamath region and $n = 2$ from 
the southern Sierra Nevada) for linkage analysis. For each of these trees, \SIrange{75}{100}{} 
seeds were germinated and planted in the common garden. Upon germination, haploid 
megagametophyte tissue was rescued from each growing seedling, cleaned, and stored for further 
analysis in \SI{1.5}{\mL} Eppendorf tubes at \SI{-20}{\celsius}.


\subsection{Library Preparation and Sequencing}

Total genomic DNA was isolated from each rescued megagametophyte using the DNeasy 96 Plant 
kit following the manufacturer’s protocol (Qiagen, Germantown, MD). Restriction site associated DNA 
sequencing (RADseq, see Davey and Blaxter 2010) was used to generate a genome-wide set of 
single nucleotide polymorphism (SNP) markers for linkage mapping following the protocol 
outlined by Parchman et al. (2012). In brief, this protocol is a double digestion RADseq 
approach based on digestion of total genomic DNA using EcoR1 and MseI followed by single-end 
sequencing on the Illumina HiSeq 2000 (or 2500) platform. Following digestion, adaptors 
containing amplification and sequencing primers, as well as barcodes for multiplexing, 
were ligated on to digested DNA fragments. Here, we chose to multiplex 96 samples using the 
barcodes available from Parchman et al. (2012). These barcodes are a mixture of 8, 9, and 
10 bp tags that differ by at least four bases, so as to accommodate sample identification in the 
presence of sequencing errors. Following ligation, successfully ligated DNA fragments were 
amplified using PCR and amplified fragments were size selected using gel electrophoresis. We selected 
fragments in the size range of 400 bp (\SIrange{300}{500}{bp}) by excising and purifying DNA from 2.5\% 
agarose gels using QIAquick Gel Extraction Kits (Qiagen).


\subsection{DNA Sequence Analysis}

\subsection{Linkage Analysis}

\section{Results}

\section{Discussion}

\section{Acknowledgements}

The authors would like to thank the staff at the USDA Institute of Forest Genetics, the 
VCU Nucleic Acids Research Facility, and the VCU Center for High Performance Computing. 
In addition, we would like to thank Tom Blush and XXX Tricia’s husband for help in obtaining 
seeds. Funding for this project was made available to AJE via startup funds from Virginia 
Commonwealth University. CJF was supported under NSF NPGI: Postdoctoral Research Fellowship 
in Biology FY 2013 Award \#IOS-1306622.

\end{document}
